Source/Core/Core/HW/Wiimote.h

```diff
@@ -45,7 +47,7 @@ enum
   WIIMOTE_SRC_REAL = 2,
 };
 
-extern unsigned int g_wiimote_sources[MAX_BBMOTES];
+extern std::atomic<unsigned int> g_wiimote_sources[MAX_BBMOTES];
```

Comment:
```suggestion
extern std::atomic<u32> g_wiimote_sources[MAX_BBMOTES];
```

to make it consistent with its definition. This could also technically be a `std::array`, but since it's pre-existing I'll leave that call up to you.

Reply:
Done. :)